SocialFilter/sections/specific.txt

```diff
@@ -46,6 +46,7 @@ gpmradio.ru,originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,ana
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+rambler-co.ru##.share__inner
```

Comment:
Why did you add this? Shouldn't this already covered by `##.share`?

Reply:
I made a mistake. sorry.